packages/expo-video-thumbnails/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### 💡 Others
 
+## 6.2.0 — 2021-12-14
+
+- Thumbnails can now be generated with 'content://' paths on Android[#15553](https://github.com/expo/expo/pull/15553) by (@lukebrandonfarrell)[https://github.com/lukebrandonfarrell]
```

Comment:
```suggestion
- Thumbnails can now be generated with `content://` paths on Android. ([#15553](https://github.com/expo/expo/pull/15553) by [@lukebrandonfarrell](https://github.com/lukebrandonfarrell))
```

Reply:
Thanks will fix this! Also need to fix tests by not changing version in package.json